config/install/field.storage.node.field_vehicle_.yml

```diff
@@ -0,0 +1,20 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - node
+id: node.field_vehicle_
+field_name: field_vehicle_
```

Comment:
This field's machine name doesn't match the content model.
![vehicle_field_vehicle_class_machine_name](https://github.com/notS0Solo/swapimigration/assets/1434979/fb85fa5e-5f04-40d3-a9d8-f75efabb29bc)

Reply:
Fixed